README.md

```diff
@@ -47,6 +47,8 @@ chmod +x ./kind
 mv ./kind /some-dir-in-your-PATH/kind
 ```
 
+## Usage
+
 To use kind, you will need to [install docker].  
```

Comment:
this sentence seems more relevant to the installation section.

Reply:
Fixed!